src/libAtomVM/defaultatoms.c

```diff
@@ -43,6 +43,7 @@ static const char *const case_clause_atom = "\xB" "case_clause";
 static const char *const if_clause_atom = "\x9" "if_clause";
 static const char *const throw_atom = "\x5" "throw";
 static const char *const low_entropy_atom = "\xB" "low_entropy";
+static const char *const unsupported_atom = "\xC" "unsupported";
```

Comment:
`unsupported` is 11, therefore 0xB instead of 0xC

Reply:
My bad.  Even worse, this part of the PR "escaped" from a branch I took this code from.  I thought I needed it for the `is_float/1` check (to raise an error), but it turns out that raising an error from a guard is not a good idea :)

Do you think I should just remove the `unsupported` atom, or should we keep it?  It is used in the bit syntax PR, as there are some use-cases that code does not support, and I wanted to raise errors with that specific atom.  Another alternative atom name is `notsup`, which I know is used in the OTP `crypto` application, but I don't know if it used elsewhere in OTP.  I am open to anything.  In the meantime, I have fixed the prefix len.

Reply:
let's merge this, and we'll think about it later.